### DISCUSSION TITLE
pip install open-webui Not working

### DISCUSSION THREAD

**Discussion by MominIqbal-1234** [open-webui/open-webui#7315] [General]
python version : 3.12.3

pip install open-webui Not working

# Bug Report

![image](https://github.com/user-attachments/assets/0ec457da-d1e1-4acf-acbf-06b5737481cd)

**Comment by deshpandeamith** [comment]
I ran into similar issue and I had old version of python (9+) in my environment path (on Mac). I removed the python from path and used my local python 12+ version working using profile (make sure "python --version" command works and has version required for open-webui). pip install open-webui would work
Also after removing the path and setting local environment, make sure you start a new terminal and test the python version to confirm before installing open-webui.
Hope it helps
